docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 100
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Prometheus.
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. The HTTP Prometheus server URL field is the address of your Prometheus server:
+
+   - When running locally or with Docker using port mapping,
+     the address is likely `http://localhost:9090`.
+
+   - When running with docker-compose or Kubernetes,
+     the address is likely `http://prometheus:9090`.
+
+1. To see metrics, select **Explore**.
+1. Select a metric, apply (optional) label filters, and select **Run query**.
+1. Learn more about PromQL and querying by reading [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/).
```

Comment:
Sure. I think it might still be beneficial to split the steps here (the first four are once-off steps, the last three steps are the steps you need to follow each time).

Reply:
I have attempted to split these two procedures into two.